gwt-localizable-processor/src/main/java/org/gwtproject/i18n/processor/ResourceEntry.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright © 2018 The GWT Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gwtproject.i18n.processor;
+
+import java.util.Collection;
+
+/** Definition of a single entry for a resource. */
+public interface ResourceEntry {
```

Comment:
possibly delete? might need to see if having something more generic like this helps with messages

Reply:
deletd